cmake-tool/helpers/application_settings.cmake

```diff
@@ -96,79 +96,93 @@ endfunction()
 #
 # Calling this function will result in forced updates to the cache.
 function(correct_platform_strings)
-    set(_REWRITE ON)
+
     set(
         correct_platform_strings_platform_aliases
-        sabre
-        wandq
-        kzm
-        rpi3
-        exynos5250
-        exynos5410
-        exynos5422
-        am335x-boneblack
-        am335x-boneblue
-        x86_64
-        ia32
+        "-KernelARMPlatform"
+        "imx6,sabre,wandq"
+        "imx31,kzm"
+        "bcm2837,rpi3"
+        "exynos5,exynos5250,exynos5410,exynos5,exynos5422,exynos5"
```

Comment:
Are you able to add a comment describing the new format of this list?
I'm a bit confused about how to interpret the exynos line.

Reply:
Thanks for the review, all "exynos5" except the first one are to be removed. That's a left over from an earlier attempt. Will add a comment describing the format.